src/guide/advanced/isi.md

```diff
@@ -81,41 +99,49 @@ are assumed to be non-negative as well, so you can never have `-1.0` of
 ### Grant/Revoke
 
 These are used specifically for permissions. When minting a permission
-token, a user is granted `X` amount of permissions to do `Y` which will
-expire once all `X` operations were performed.
+token, a user is granted `X` amount of permissions to do `Y`, and the token
+will expire once all `X` operations were performed.
 
-By contrast, a `Grant` operation can be used to grant either a single
-permission, or a group of permissions (or a "role"), to a user permanently.
-As such these instructions should be used carefully.
+By contrast, the `Grant` operation can be used to permanently grant a user
+either a single permission, or a group of permissions (a "role"). As such
+these instructions should be used carefully.
 
 ### `SetKeyValue`/`RemoveKeyValue`
 
-These instructions are used with the key/value store asset type. This use
+These instructions are used with the key/value `Store` asset type. This use
 case has not received much attention so far, because storing data in the
-blockchain is rather advanced topic that we shall cover separately.
+blockchain is a rather advanced topic that we shall cover separately.
 
 ### Query
 
-These are all the queries that can be made from the client side. This is
-not necessarily the only kind of information that is available on the
-network, but it's the only kind of information that is guaranteed to be
-accessible on all networks. Telemetry data is optional to the specific
-deployment of Iroha. Access to your account balance is required function.
```

Comment:
tbh this part seemed a bit out of place here. It says "These are all the queries...", and I'm not sure what "these" refer to, I've tried working around this but I'm not sure I got it right

Reply:
I moved the queries into their own file, but forgot to remove the reference. You did the right thing.